__tests__/integrations/php/laravel.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine
+FROM alpine:3.16
```

Comment:
Tried changing this to a bare `php:8.1` image but ran into a bunch of problems with Composer not running right so Alpine is fine.